.gitignore

```diff
@@ -1,6 +1,6 @@
 composer.phar
 vendor/
-
+.idea*
```

Comment:
Whoa! I honestly didn't know (or forgot) about a global gitignore file. This is news to me and as Eric said, noted.  

I can go either way.

Thanks for the input.